### ISSUE TITLE
Display built-in plugins

### ISSUE THREAD

**Issue by vitaliy-guliy** [eclipse/che-theia#244]
### Description
 

Display built-in (default) plugins in the list of installed plugins. We have to indicate somehow that plugin is default ( provided by Theia editor docker image ).
Default plugins are always present on the file system and cannot be removed.

### Reproduction Steps

**OS and version:**    

**Diagnostics:**

**Comment by vitaliy-guliy** [comment]
I have already did some work on this issue and included plugins in the list. But there is still a lot of work on plugin decorations.
![Screenshot from 2019-05-24 10-30-59](https://user-images.githubusercontent.com/1655894/58310380-2d3b9700-7e0f-11e9-8846-225cabf98bb8.png)

**Comment by sunix** [comment]
@vitaliy-guliy i don't think we want to display these plugins ... not critical for customers

**Comment by benoitf** [comment]
I think it's useful to see them and to not have a way to install/uninstall them

it's like default apps on Android or Extensions in VS Code, there are there but you can't uninstall them
